app/param_filters.go

```diff
@@ -40,7 +40,7 @@ func bankParamFilter(msg sdk.Msg) error {
 	return nil
 }
 
-// consensusParamstakingParamFilterFilter checks if the parameters in the MsgUpdateParams ensures that only valid changes are allowed.
+// stakingParamFilter checks if the parameters in the MsgUpdateParams ensures that only valid changes are allowed.
```

Comment:
Can you please fix the grammar in this comment at the same time?

```suggestion
// stakingParamFilter ensures that the parameters in the MsgUpdateParams contain only valid changes.
```